python/cugraph/comms/comms.py

```diff
@@ -114,6 +174,10 @@ def get_worker_id(sID):
     return sessionstate['wid']
 
 
-def get_n_workers(sID):
-    sessionstate = worker_state(sID)
-    return sessionstate['nworkers']
+def get_n_workers(sID=None):
```

Comment:
I could sware I saw several ways in the python layer that already get the number of workers/GPUs:
- get_n_workers() (from read_utils)
- len(get_visible_devices())
- len(numba.cuda.gpus)

 We should document why this one is different/better or avoid multiplying code doing the same thing.

Reply:
Agreed, I'll look into this for the next commit.